Gruntfile.js

```diff
@@ -325,14 +267,11 @@ module.exports = function(grunt) {
   ]);
 
   grunt.registerTask("generateJS", [
-    "concat:plottable",
     "concat:svgtypewriter",
-    "concat:definitions",
     "sed:definitions",
```

Comment:
Nothing should blow up since there is no longer a reference to the `build/` directory.